db/migrate/20160203102911_migrate_users_to_have_role.rb

```diff
@@ -0,0 +1,23 @@
+class MigrateUsersToHaveRole < ActiveRecord::Migration
+  def up
+    Comfy::Cms::User.all.each do |user|
+      user.update_attribute(:role, new_role_for(user))
+    end
+  end
+
+  def down
+    Comfy::Cms::User.all.each do |user|
+      user.update_attribute(:admin, user.admin?)
+    end
+  end
+
+  private
+
+  def new_role_for(user)
+    if user.read_attribute(:admin)
+      Comfy::Cms::User.roles[:admin]
+    else
+      Comfy::Cms::User.roles[:user]
+    end
+  end
+end
```

Comment:
I would move this to a rake task and delete this migration and the next one (removal of admin field). ˆThen after 'everybody' is happy and the data is migrated, you can delete the admin field in another deploy, but doing in steps. But  if you think that do at once will work fine, then is ok. Wdyt?